### PR TITLE
CRF-1: Initial commit

### PR DESCRIPTION
**Pull request by paulridout**
### JIRA link (if applicable) ###
CRF-1 (https://tools.hmcts.net/jira/browse/CRF-1)

### Change description ###
- Ran init.sh script to perform initial setup of repository.
- Removed .github/workflows/ci.yml as it just duplicates the Jenkins pipeline.
- Reverted Dockerfile to use Java 17.  Only this file had been updated to use Java 21 in the spring-boot-template
- Updated product and component fields in Jenkinsfile_nightly as they weren't changed by init.sh.
- Removed default CODEOWNERS file.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/civil-rtl-export/Chart.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 apiVersion: v2
 appVersion: "1.0"
-description: A Helm chart for spring-boot-template App
-name: rpe-spring-boot-template
-home: https://github.com/hmcts/spring-boot-template
+description: A Helm chart for civil-rtl-export App
+name: civil-civil-rtl-export
```
Comment (kiran-yenigala-hmcts) [inline charts/civil-rtl-export/Chart.yaml:4]: can this be: civil-rtl-export

File: src/main/java/uk/gov/hmcts/reform/civil/controllers/RootController.java [hunk 2]
```diff
@@ -23,6 +23,6 @@ public class RootController {
      */
     @GetMapping("/")
     public ResponseEntity<String> welcome() {
-        return ok("Welcome to spring-boot-template");
+        return ok("Welcome to civil-rtl-export");
```
Comment (kiran-yenigala-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/controllers/RootController.java:26]: change to : Welcome to Civil Rtl Export

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -12,13 +12,13 @@ management:
         include: health, info, prometheus
 
 springdoc:
-  packagesToScan: uk.gov.hmcts.reform.demo.controllers
+  packagesToScan: uk.gov.hmcts.reform.rtl-export.controllers
```
Comment (kiran-yenigala-hmcts) [inline src/main/resources/application.yaml:15]: is this package exists: uk.gov.hmcts.reform.rtl-export

**Review by kiran-yenigala-hmcts — CHANGES_REQUESTED** [review]
Comments added

File: src/main/resources/application.yaml (hunk 3 above)
Comment (paulridout) [reply src/main/resources/application.yaml]: Changed packagesToScan to "uk.giv.hmcts.refrom.civil.controllers".

**Review by kiran-yenigala-hmcts — APPROVED** [review]

**Pull request merged**